bugzilla2gitlab/models.py

```diff
@@ -90,6 +91,21 @@ def create_labels(self, component, operating_system):
 
         self.labels = ",".join(labels)
 
+    def create_milestone(self, title):
+        '''
+        Looks up milestone id given its title or creates a new one. Does nothing if milestones
+        are not mapped.
+        '''
+        if not conf.map_milestones or title in conf.milestones_to_skip:
```

Comment:
I think this check should happen outside of the method, such that the method would not be called if these conditions aren't satisfied.